main.go

```diff
@@ -29,7 +29,7 @@ type Client struct {
 func NewClient(token string) *Client {
 	return &Client{
 		token:      token,
-		httpClient: &http.Client{},
+		httpClient: NewRetryableClient(),
```

Comment:
this requires change the signature from HTTPClient.Do() which is public for this package, so we would potentially break backwards compatibility. Without changing the signature of Do() we can just call NewRetryableClient().StandardClient()  here instead. 
The SDK should probably support a way to override the default http client in the long run :thinking:

Reply:
Nice change, thanks!